src/utils/setup.ts

```diff
@@ -157,27 +158,29 @@ const runMigrations = async () => {
 
 const updateTotalMsgPerTribe = async () => {
   try {
-    const result = (await sequelize.query(
-      `
-      SELECT * FROM sphinx_contacts
-      INNER JOIN sphinx_chats
-      ON sphinx_contacts.public_key = sphinx_chats.owner_pubkey
-      INNER JOIN sphinx_chat_members
-      ON sphinx_chats.id = sphinx_chat_members.chat_id
-      WHERE sphinx_contacts.is_owner = 1`,
-      {
-        model: models.ChatMember,
-        mapToModel: true, // pass true here if you have any mapped fields
-      }
-    )) as ChatMemberRecord[]
-
-    if (result.length > 0 && result[0].totalMessages === null) {
-      for (let i = 0; i < result.length; i++) {
-        const member = result[i]
-        const totalMessages = await models.Message.count({
-          where: { sender: member.contactId, chatId: member.chatId },
-        })
-        await member.update({ totalMessages })
+    const contacts = (await models.Contact.findAll({
+      where: { isOwner: true },
+    })) as ContactRecord[]
+    for (let i = 0; i < contacts.length; i++) {
+      const contact = contacts[i]
+      const tribes = (await models.Chat.findAll({
+        where: { ownerPubkey: contact.publicKey },
+      })) as ChatRecord[]
+      for (let j = 0; j < tribes.length; j++) {
+        const tribe = tribes[j]
+        const tribeMembers = (await models.ChatMember.findAll({
+          where: { chatId: tribe.id },
+        })) as ChatMemberRecord[]
+        for (let k = 0; k < tribeMembers.length; k++) {
```

Comment:
Last one use for each instead of c style loop

Reply:
disagree... for loop is "async aware"... forEach will be faster but could also overflow the SQLITE connection and cause a crash with many members (all queries would be running simultaneously)